.github/workflows/cifar_benchmark.yaml

```diff
@@ -132,6 +132,7 @@ jobs:
           python3 ./benchmarks/convert_cifar.py --model-name "16-bits-trained-v0"
 
       - name: Upload results
+        if: ${{ github.repository == 'zama-ai/concrete-ml-internal' }}
```

Comment:
why should we only upload results from the private repo ?

Reply:
The public repository doesn't have the secret needed to updload and we don't use this DB anymore.

If we don't skip this in the public repository this fails the action.